### PR TITLE
Redux Implementation

### PR DESCRIPTION
**Pull request by MatiasMMaldon01**
This pull request contains the redux pattern implementation for handling both login and shop cart.

### CODE REVIEW

File: mobile-ys/src/app/app.component.ts [hunk 1]
```diff
@@ -1,15 +1,20 @@
...
 })
 export class AppComponent {
-    isLogged: boolean = true
-    constructor() {}
+    isLogged: boolean 
+    constructor(private authService: AuthService, private store: Store<any>) {
+        this.isLogged = this.authService.loggedIn();
+    }
 
     logOut = () => {
```
Comment (camilosanchezdev) [inline mobile-ys/src/app/app.component.ts:17]: I can't find this button in the interface.

File: mobile-ys/src/app/pages/home/home.page.ts [hunk 2]
```diff
@@ -41,4 +46,7 @@ export class HomePage implements OnInit, OnDestroy {
             })
         )
     }
+    getClient():void{
```
Comment (camilosanchezdev) [inline mobile-ys/src/app/pages/home/home.page.ts:49]: The users Must can see the products on the home page without log in. Fix the interceptor service, remove navigation to Login.

**Review by camilosanchezdev — CHANGES_REQUESTED** [review]
Excellent NgRx implementation. Fix the issues and merge it.

File: mobile-ys/src/app/pages/home/home.page.ts (hunk 2 above)
Comment (MatiasMMaldon01) [reply mobile-ys/src/app/pages/home/home.page.ts:49]: Okey, i'll see it.

File: mobile-ys/src/app/app.component.ts (hunk 1 above)
Comment (MatiasMMaldon01) [reply mobile-ys/src/app/app.component.ts:17]: mmm sorry, but i don't get it.

Comment (camilosanchezdev) [reply mobile-ys/src/app/app.component.ts:17]: Mirá te muestro, cuando me logeo, no se ve el botón de Logout.

imagen

**Pull request merged**